EpikCord/client/client.py

```diff
@@ -0,0 +1,7 @@
+from ..flags import Intents
+
+class Client:
+    def __init__(self, token: str, intents: Intents):
+        self.token: str = token
```

Comment:
Not sure  that the token should be keep in the client class like this. Not even sure it should be passed to the client at construction too

Reply:
What makes you say that?

Reply:
> What makes you say that?

Well the token should be handled by something lower-level like the httpclient ig